game-core/src/test/java/org/triplea/game/server/ban/check/PlayerDisconnectActionTest.java

```diff
@@ -0,0 +1,108 @@
+package org.triplea.game.server.ban.check;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import games.strategy.net.INode;
+import games.strategy.net.IServerMessenger;
+import games.strategy.net.Node;
+import java.net.InetSocketAddress;
+import java.util.Set;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.domain.data.PlayerName;
+
+@ExtendWith(MockitoExtension.class)
+class PlayerDisconnectActionTest {
+  private static final INode NODE_1 =
+      new Node("node-1", InetSocketAddress.createUnresolved("1.1.1.1", 5000));
```

Comment:
Side-Note:
1.1.1.1 is an actual IP that's being used. See https://1.1.1.1

Reply:
Thanks, probably better to keep the value to be a true dummy value: 0cb3424bd